apps/settings/elements/call.html

```diff
@@ -61,25 +61,24 @@ <h1 data-l10n-id="callSettings-header"></h1>
 
     <form role="dialog" data-type="confirm" hidden class="cw-alert">
       <section>
-        <h1 data-l10n-id="confirmCallWaitingTitle">Unable to confirm call waiting preferences</h1>
+        <h1 data-l10n-id="confirmCallWaitingTitle"></h1>
         <p>
-          <strong data-l10n-id="confirmCallWaitingDesc">The device is currently unable to communicate with the carrier. Try setting the preference again.</strong>
+          <strong data-l10n-id="confirmCallWaitingDesc"></strong>
         </p>
         <p class="cw-alert-sub-p">
           <label class="pack-switch cw-alert-checkbox-label">
             <input type="checkbox" data-ignore>
-            <span data-l10n-id="callWaiting">Call Waiting</span>
+            <span data-l10n-id="callWaiting"></span>
           </label>
         </p>
       </section>
       <menu>
-        <button class="cw-alert-cancel" data-l10n-id="cancel">Cancel</button>
```

Comment:
There are still texts under the caller id selector. Would you mind remove it while you are on it?